### ISSUE TITLE
Search timeout needs too long to return when searching partitions after releasing and load a partition

### ISSUE THREAD

**Issue by binbinlv** [milvus-io/milvus#6997]
#### Steps/Code to reproduce:
Run test case:
/milvus/tests20/python_client/testcases/test_search.py::TestCollectionSearch::test_search_partition_after_release_load

#### Expected result:
No long timeout is needed

#### Actual results:
Need long timeout to return

#### Environment:
- Milvus version():ad77a6e
- Deployment mode(standalone or cluster):standalone
- SDK version(e.g. pymilvus v2.0.0rc2):
Name: pymilvus
Version: 2.0.0rc3.dev16
Name: pymilvus-orm
Version: 2.0.0rc3.dev15

- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:

**Comment by DragonDriver** [comment]
working on this
/assign

**Comment by DragonDriver** [comment]
/assign @xige-16 
PTAL

**Comment by bigsheeper** [comment]
Did you reproduce this issue? @DragonDriver

**Comment by binbinlv** [comment]
Yes, each nightly case run have this issue.

**Comment by binbinlv** [comment]
And this issue only appears when running all the search test cases (test_search.py) (e.g., even set big timeout = 1000), it will pass if only run this test case separately even set small timeout (e.g., timeout = 20).

**Comment by bigsheeper** [comment]
> And this issue only appears when running all the search test cases (test_search.py) (e.g., even set big timeout = 1000), it will pass if only run this test case separately even set small timeout (e.g., timeout = 20).

This case failed in my local computer, and maybe I'v found the reason, but did you mean it can search successfully on your tests?

**Comment by binbinlv** [comment]
Yes, if I only run this case, it will pass. And when I run all the search test cases, it will fail at this test case.
@bigsheeper

**Comment by bigsheeper** [comment]
I'v run this test, and it get failed. The log shows that the search requset happened before the load request. The reason is the loading percentage of old meta info from first loading is still 100% even though called the release partition requset. So query coord needs to check the loading state correctly in the case of partitions loading or releasing.

I'll fix this later.

**Comment by binbinlv** [comment]
OK， thanks.

**Comment by yanliang567** [comment]
mark as critical as it blocks ci nightly: https://ci.milvus.io:18080/jenkins/blue/organizations/jenkins/milvus-nightly-ci/detail/master/95/pipeline/126

**Comment by czs007** [comment]
@bigsheeper  is working on this.  Have figured out the reason.
